slam-libraries/viam-cartographer/src/slam_service/config.cc

```diff
@@ -69,9 +71,11 @@ void ParseAndValidateConfigParams(int argc, char** argv,
     LOG(INFO) << "data_rate_ms: " << FLAGS_data_rate_ms;
     LOG(INFO) << "map_rate_sec: " << FLAGS_map_rate_sec;
     LOG(INFO) << "delete_processed_data: " << FLAGS_delete_processed_data;
+    LOG(INFO) << "use_live_data: " << FLAGS_use_live_data;
 
     slamService.path_to_data = FLAGS_data_dir + "/data";
     slamService.path_to_map = FLAGS_data_dir + "/map";
+    slamService.offline_flag = !(FLAGS_use_live_data);
```

Comment:
Do we want to add in an error if online mode is given with an empty list of sensors (and should this also be initially checked in RDK?)

Reply:
i think only having the check in rdk is fine, as the slam algorithms should never run into that test case

Reply:
I agree and have left a similar comment to that affect in the RDK PR as the issue of erroring out was not covered in the scope doc

Reply:
We've been a little inconsistent in what validation occurs in orb_grpc_server/carto_grpc_server. I would say that parameter validation *must* occur rdk. In some cases, we've additionally chosen to include validation in orb_grpc_server/carto_grpc_server, primarily when the program cannot continue without it (e.g. a missing required parameter). My suggestion is to avoid logical validation in orb_grpc_server/carto_grpc_server (e.g. a disallowed combination of parameters), since this duplicates validation in rdk. However, I don't feel so strongly about this, particularly since we will eventually combine the slam service with orb_grpc_server/carto_grpc_server.